app/assets/stylesheets/components/buttons.scss

```diff
@@ -7,12 +7,3 @@
     border-color: #477958;
   };
 }
-
-.btn-blue {
-  @include btn-solid($text-white, #599bcd, #3072b3);
-  border-color: #2a65a0;
-}
-
-.btn-large {
```

Comment:
It appears this CSS is still being used in `app/views/organizations/show.html.erb`
https://github.com/education/classroom/blob/10b4c4a2254aac8c637974d3a8ce17258e858651/app/views/organizations/show.html.erb#L23